pages/index.tsx

```diff
@@ -93,9 +71,11 @@ const Home: NextPage<
           </p>
         </CardLink>
 
-        <div className={homeStyles.featured_pokemon}>
-          <PokemonCard {...pokemon} />
-        </div>
+        {pokemon && (
```

Comment:
Apparently, the API also doesn't ensure that the pokemon get's returned. SO it seems like the codegen tool is helping us avoid potential bugs, by ensuring we have a pokemon before rendering it!